guarddog/analyzer/sourcecode/dll-hijacking.yml

```diff
@@ -34,6 +34,24 @@ rules:
                 # a string with .dll or .so in it
                 - pattern: "..."
                 - pattern-regex: (?i).*?\.(dll|so)
+        - patterns:
+          - pattern: "..."
+          - pattern-either:
+            # See MITRE ATT&CK "System Binary Proxy Execution" techniques
+            # https://attack.mitre.org/techniques/T1218/
```

Comment:
I would use (?i) in front of all regex to make the case insensitve

Reply:
we could move this entire pattern-either block below line 22

Reply:
Both changes made